tests/Unit/CartTest.php

```diff
@@ -4,18 +4,52 @@
 
 class CartTest extends TestCase
 {
+    /**
+     * If item is added to the cart
+     *
+     * @covers Cart::getItems
+     */
     public function testAddItem()
     {
         $cart = new Cart();
-
         $cart->addItem('foo');
         $this->assertEquals(array('foo'), $cart->getItems());
         $this->assertNotEquals(array('bar'), $cart->getItems());
     }
-    public function testGetItems()
+
+    /**
+     * Remove an item from the cart
+     *
+     * @covers Cart::getItems
+     */
+    public function testRemoveItem()
     {
         $cart = new Cart();
+        $cart->removeItem('foo');
+        $this->assertEmpty('', $cart->getItems());
```

Comment:
I'm going to assume setting the `assertEmpty` method to this is somewhat cheating? 😁